objects/client.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require_relative '../service/file_handler'
+require_relative '../service/language'
+
+# Description/Explanation of Client Class
+class Client
+  include Language
+  attr_reader :client
+
+  def initialize(clients)
+    @clients = converted_file(clients)
+    @client = {}
+    @answer = ''
+    @full_name = ''
+    @id = 0
+  end
+
+  def converted_file(file)
+    file.each { |hash| hash['id'] = hash['id'].to_i }
+  end
+
+  def locate_client
+    @clients.find { |client| "#{client['first_name']} #{client['last_name']}" == @full_name }
+  end
+
+  def info
```

Comment:
Please, refactor it, I mean integrate all conditions

Reply:
Please give me some more details regarding this comment. What do you mean by 'all conditions'?